imageregistry/v1/types.go

```diff
@@ -333,6 +386,9 @@ type ImageRegistryConfigStorage struct {
 	// ibmcos represents configuration that uses IBM Cloud Object Storage.
 	// +optional
 	IBMCOS *ImageRegistryConfigStorageIBMCOS `json:"ibmcos,omitempty"`
+	// OSS represents configuration that uses Alibaba Cloud Object Storage Service.
```

Comment:
These comments will be visible by customers and should contain YAML names, so
```suggestion
	// oss represents configuration that uses Alibaba Cloud Object Storage Service.
```